samples/Server/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "module": "commonjs",
     "esModuleInterop": true,
     "target": "es6",
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Any specific reason that we change it here? It will weaken the type check for the folder, if we miss any type let's either declare them or even force cast them

Reply:
i remember there were some issues when i created a type declaration myself, but let me reproduce it and will let you know why it didn't work for me

Reply:
thanks, i have updated my code and noImplicitAny is set back to `true` now

Reply:
Can we just remove the `noImplicitAny` entirely?

This was put in as a workaround due to some other issues found a while ago. The correct setting is inherited from using `strict mode` and we should not need this parameter.

Reply:
i am not sure, `noImplicitAny` is set for all packages in this repo, do we want to remove it everywhere?